helm/aws-pod-identity-webhook/templates/Deployment.yaml

```diff
@@ -24,6 +24,16 @@ spec:
         {{- include "labels.common" . | nindent 8 }}
         component: webhook
     spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app.kubernetes.io/name
+                    operator: In
+                    values:
+                      - {{ include "name" . | quote }}
+              topologyKey: "kubernetes.io/hostname"
```

Comment:
The linter was bugging me about this. Please make sure to review it just in case I screwed it up.

Reply:
isn't the `- labelSelector` part with wrong indent? https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity

Reply:
You mean that it needs to be in the same column as `requiredDuringSchedulingIgnoredDuringExecution`? It's one of the items in the `requiredDuringSchedulingIgnoredDuringExecution` array, so it doesn't matter if it's indented or not AFAIK.